app/views/sudo_mode/reauthenticate.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1>Confirm Access</h1>
+
+<p>Signed in as <%= current_user.email %></p>
+
+<%= form_tag(request.path, method: request.request_method_symbol) do %>
+  <%
+    Rack::Utils
+      .parse_query(request.request_parameters.to_query)
```

Comment:
I think we could make this a partial that we include in all the forms rendered from the `LoginsController` and then have some shared logic in `LoginsController#complete` to handle input from that.